### PR TITLE
test: convert wd method calls to webdriverio method calls

### PR DESCRIPTION
**Pull request by mwakizaka**
- only in `test/functional/basic/basic-e2e-specs.js`
- if no problem, I'd like to continue working on the others under  `test/functional` as well.
- some of the methods are jswp (e.g. `getLogTypes`). is it ok?

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Thank you!

You need to remove comment out from https://github.com/appium/appium-xcuitest-driver/blob/master/ci-jobs/templates/xcuitest-e2e-template.yml#L13-L22 to run the fixed tests. Then, you can comment out other tests in the `test/functional/basic`

**Comment by mwakizaka** [comment]
Configured as https://github.com/appium/appium-xcuitest-driver/pull/1447/commits/a2b6cf54b31ac2676dc15700fb7cdec2d9c20632
In this PR is it better to update all wd usages under `test/functional/basic/` or update only `basic-e2e-specs.js` for ease of review?

**Comment by KazuCocoa** [comment]
One test file update seems great

**Review by KazuCocoa — APPROVED** [review]
We need to check `process.env.CLOUD` flag, otherwise this PR itself seems good to me to convert wd  to webdriverio

File: test/functional/basic/basic-e2e-specs.js [hunk 1]
```diff
@@ -92,6 +88,17 @@ describe('XCUITestDriver - basics -', function () {
       actual.viewportRect.width.should.be.a('number');
       delete actual.viewportRect;
 
+      // convert w3c caps into mjswp caps
+      let mjswpCaps = {};
```
Comment (mykola-mokhnach) [inline test/functional/basic/basic-e2e-specs.js:92]: there should be no occurrences of deprecated capabilities. We should only use w3c stuff

Comment (mykola-mokhnach) [reply test/functional/basic/basic-e2e-specs.js:92]: actually, getSession API itself is not a w3c one 🤷

Comment (mwakizaka) [reply test/functional/basic/basic-e2e-specs.js:92]: right, but I couldn't find a way to obtain w3c capabilities from appium server. I can continue working on this tests if you provide an alternative approach.

Comment (mykola-mokhnach) [reply test/functional/basic/basic-e2e-specs.js:92]: maybe we don't need this test at all...
Let's keep it as is for now and leave a TODO there

Comment (mwakizaka) [reply test/functional/basic/basic-e2e-specs.js:92]: I see, thanks. BTW, it is ok to use mjsonwp and appium (`/session/:sessionId/appium/`) APIs?

Comment (mykola-mokhnach) [reply test/functional/basic/basic-e2e-specs.js:92]: Eventually I would love to deprecate all non-w3c endpoints and move them under `mobile:`-like extensions, so the `external` REST spec is 100% compatible with webdriver standard.

**Pull request merged**